content/docs/manual/plugin-storages.md

```diff
@@ -12,21 +12,56 @@ It allows you at runtime to dynamically add/remove backends and storages. More d
 When adding a backend, a short name of the backend library can be provided instead of the complete path to the library file.
 In such case the Storages plugin will search and load the library file with such name:
  - on Unix/Linux: `libzbackend_<name>.so`
- - on MacOS: `libzbackend_<name>.dylib`
+ - on macOS: `libzbackend_<name>.dylib`
  - on Windows: `zbackend_<name>.dll`
 
 The list of paths in which the Storages plugin will search for backends can be configured via the `--backend-search-dir` option (that can be repeated to specify several directories). The default list can be seen in using the `--help` option of the zenoh router.
 
-Note that by default the Storages plugin already embeds and provides at startup the [Memory backend](../backends-list#memory-backend).
+Note that by default the Storages plugin already embeds the [Memory backend](../backends-list#memory-backend).
 
 ------
 **Library name:** `zplugin_storages`
 
 ------
+## Until v0.5
 **Startup arguments** (added to zenoh router's startup arguments):
- - `--backend-search-dir=[DIRECTORY]` : A directory where to search for backends libraries to load.
-                             Repeat this option to specify several search directories.
- - `--no-backend`: If true, no backend (and thus no storage) are created at startup.
-                   If false (default) the Memory backend it present at startup.
- - `--mem-storage=[PATH_EXPR]` : A memory storage to be created at start-up. Repeat this option to created several storages.
+- `--backend-search-dir=[DIRECTORY]` : A directory where to search for backends libraries to load.
+                            Repeat this option to specify several search directories.
+- `--no-backend`: If true, no backend (and thus no storage) are created at startup.
+                  If false (default) the Memory backend is present at startup.
+- `--mem-storage=[PATH_EXPR]` : A memory storage to be created at start-up. Repeat this option to create several storages.
 
+## From v0.6
+Storages now use structured [configurations](../configuration) to require the creation and deletion of backends and storages.  
+The main schema is as follows:
+```typescript
```

Comment:
Is this syntax a standard for JSON schemes ?
I think an example of a working JSON file might also help user to understand the schema.

Reply:
I can point back to `EXAMPLE_CONFIG.json5` here too, but the typescript format strikes a nice balance of describing types instead of values, and being intuitive to a large amount of people.